src/apps/VirtualApp/VirtualChat.js

```diff
@@ -180,29 +191,69 @@ class VirtualChat extends Component {
       success: () => {
         Janus.log(':: Message sent ::');
         this.setState({ input_value: '' });
-        if (!room_chat) {
+        if (room_chat === TABS.SUPPORT){
           support_msgs.push(msg);
           this.setState({ support_msgs });
+        }else{
+          messages.push(msg);
```

Comment:
You don't want to push your own message when we are in CHAT, right?
} else if (room_chat === TABS.QUESTIONS) {
  messages.push(msg)